go.mod

```diff
@@ -19,21 +19,21 @@ require (
 	github.com/go-sql-driver/mysql v1.6.0 // indirect
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/google/uuid v1.3.0 // indirect
-	github.com/gorilla/websocket v1.4.2 // indirect
-	github.com/hashicorp/go-hclog v1.1.0 // indirect
+	github.com/gorilla/websocket v1.5.0 // indirect
```

Comment:
why is there a go.mod, and why is this PR changing it?

Reply:
This PR shouldn't change go.mod from what I understand. Although I may have habitually ran `go mod tidy`.
The dev docs use some code to pull godoc from the mattermost-server sources, hence the `go.mod`.
I'll double-check what's happened and revert the change.